4.analyze_data/README.md

```diff
@@ -2,10 +2,10 @@
 
 In this module, we perform analysis on the data to uncover answers to our goals as specified in [the main README](../README.md).
 
-
 The analyses we perform are:
 
 1. Linear modeling per feature in each plate. The beta coefficients for WT treatment contribution and cell count contribution are plotted as a scatter plot.
```

Comment:
Optional: A formula of the linear model would be nice with the terms defined?

Reply:
Hmm this is an interesting idea! I might consider this out of the scope of this PR but I want to come back to this!